demo/client/src/renderer.ts

```diff
@@ -146,7 +146,9 @@ export abstract class Renderer {
         this.clearDestFramebuffer();
 
         // Start timing rendering.
-        if (this.timerQueryPollInterval == null) {
+        if (this.timerQueryPollInterval == null &&
```

Comment:
The entire commit that makes timer queries optional shouldn't be necessary now since I made this change right before showing my slides at the Rust meetup.